README.md

```diff
@@ -126,11 +126,23 @@ notes:
 
 ### running the console
 
-to **run the console command-line tool**, simply
+to **run the console command-line tool**, install `tokio-console` from [crates.io](https://crates.io/crates/tokio-console)
+
+```shell
+$ cargo install --locked tokio-console
+```
+
+and run locally
+
 ```shell
-$ cargo run
+$ tokio-console
 ```
-in this repository.
+
+> **alternative method:** run the tool from a local checkout of this repository
```

Comment:
maybe worth noting here that this is primarily intended for developers, or for using currently-unreleased changes? not a blocker.